### PR TITLE
Update TF version required for the tf.function guide

### PR DESCRIPTION
**Pull request by Binjian**
tried on colab,
some of the features run only on tensorflow >= 2.9.1 
you might need to install the nightly built version in order to try this tutorial.

### CODE REVIEW

**Comment by Binjian** [comment]
I think I signed the CLA right now. Hope it worked.

**Comment by 8bitmp3** [comment]
CLA looks 👍 

Reviewing the issue with TF versioning on our side @Binjian Thanks for bringing this up

cc @MarkDaoust @macd @shilpakancharla

**Review by MarkDaoust — DISMISSED** [review]
Thanks for pointing this out. 

I switched it to use tf>=2.9 for more stability, and so this will be a noop once TensorFlow is upgraded in Colab.

File: site/en/guide/function.ipynb [hunk 1]
```diff
@@ -91,6 +91,8 @@
       },
       "outputs": [],
       "source": [
+        "# Update tensorflow (this notebook requires tf>=2.9)\n",
```
Comment (8bitmp3) [inline site/en/guide/function.ipynb:94]:
```suggestion
        "# Update TensorFlow, as this notebook requires version 2.9 or later\n",
```

**Review by 8bitmp3 — APPROVED** [review]

**Pull request merged**